r/tests/testthat/test-dataset-csv.R

```diff
@@ -376,7 +417,7 @@ test_that("skip argument in open_dataset", {
     header_csv_dir,
     format = "csv",
     schema = schema(int = int32(), dbl = float64()),
-    skip = 1
+    skip_rows = 1
```

Comment:
Why this change?

Reply:
Ah looked at this again and I shouldn't have changed it. It was complaining that `skip` was a readr style arg, but we were internally appending an Arrow-style arg `column_names` when we passed an explicit schema. It's a little messy, but I basically changed it so internally we choose to add `column_names` (Arrow-style) or `col_names` (readr-style) based on what other arguments were passed.